core/vm.cpp

```diff
@@ -1611,7 +1611,7 @@ class Interpreter {
 
             // Convert yaml to json and push onto json array
             ryml::Tree doc;
-            for (int i = 0; i < v.size(); ++i) {
+            for (unsigned i = 0; i < v.size(); ++i) {
```

Comment:
Probably it's better to use `size_t` instead of `unsigned`.

Reply:
I am happy to make the change but elsewhere in the file, it uses `unsigned`  and I felt cleaning up to use `std::size_t` should be separate commit. 

```
 346          for (unsigned i = 0; i < stack.size(); ++i) {
973           for (unsigned i = 0; i < args.size(); ++i) {
2217         for (unsigned i = 0; i < ast.args.size(); ++i) {
...
```
I have changed all such usage to `std:size_t`

Reply:
Tested on 
```
❯ gcc --version
gcc (GCC) 10.3.1 20210422 (Red Hat 10.3.1-1)
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
And `make test` works fine.

Reply:
Thanks for caring about consistency! Well, currently we have a mix of `unsigned` and `size_t` in this context. I'll be happy with a commit which changes all such usage to `size_t`.

Reply:
I have changed, just this file, I am sure there will be more. Perhaps in a different commit. My C++ is quite rusty now :D